docs/hyperopt.md

```diff
@@ -16,6 +16,24 @@ To learn how to get data for the pairs and exchange you're interested in, head o
 !!! Bug
     Hyperopt can crash when used with only 1 CPU Core as found out in [Issue #1133](https://github.com/freqtrade/freqtrade/issues/1133)
 
+## Install hyperopt dependencies
+
+Since Hyperopt dependencies are not needed to run a bot, they are not installed by default.
```

Comment:
```suggestion
Since Hyperopt dependencies are not needed to run the bot itself, are heavy, can not be easily built on some exotic platforms (like Raspberry PI boxes), they are not installed by default. Before you run Hyperopt, you need to install corresponding part of dependencies, as described in this section below.

!!! Note
    Follow the actions described below (i.e. install hyperopt dependencies) if you are getting the `No module named 'somemodulename'. Please ensure that the hyperopt dependencies are installed.` messages when running freqtrade hyperopt subcommand.
```

-- you may simplify my addendums...